pkg/operator/controller/ingress/nodeport_service.go

```diff
@@ -214,6 +214,7 @@ func nodePortServiceChanged(current, expected *corev1.Service) (bool, *corev1.Se
 		cmpopts.IgnoreMapEntries(func(k, _ string) bool {
 			return !managedNodePortServiceAnnotations.Has(k)
 		}),
+		cmpopts.EquateEmpty(),
```

Comment:
I removed this and the unit tests still pass. Is this expected? Were the additional unit tests expected to verify EquateEmpty()?

Reply:
When I comment this line out, `TestNodePortServiceChangedEmptyAnnotations` fails:
```
--- FAIL: TestNodePortServiceChangedEmptyAnnotations (0.00s)
    --- FAIL: TestNodePortServiceChangedEmptyAnnotations/null_to_empty (0.00s)
        nodeport_service_test.go:365: 
            	Error Trace:	/home/mmasters/src/github.com/openshift/cluster-ingress-operator/pkg/operator/controller/ingress/nodeport_service_test.go:365
            	Error:      	Should be false
            	Test:       	TestNodePortServiceChangedEmptyAnnotations/null_to_empty
    --- FAIL: TestNodePortServiceChangedEmptyAnnotations/empty_to_null (0.00s)
        nodeport_service_test.go:365: 
            	Error Trace:	/home/mmasters/src/github.com/openshift/cluster-ingress-operator/pkg/operator/controller/ingress/nodeport_service_test.go:365
            	Error:      	Should be false
            	Test:       	TestNodePortServiceChangedEmptyAnnotations/empty_to_null

```

Reply:
My fault. I was not exercising the same line in the PR addition. Apologies. 

/lgtm